docs/introduction/concepts.md

```diff
@@ -31,3 +31,27 @@ Producing and exchange olive oil and coffee, even more simplified.
 ### ... to thinking bigger.
 
 ![river flow](https://rawgit.com/valueflows/valueflows/master/release-doc-in-process/vf-river.png)
+
+### Traversing value flows
+
+Value flows can be traversed forwards ("tracing") or backwards ("tracking").  Often people use the term "provenance" when looking backwards towards the source of some resource, sometimes when a problem emerges (as in a disease outbreak from food), sometimes to know the quality of the resource (as in wanting local humanely produced food with lower ecological impact).
```

Comment:
@bhaugen in https://github.com/valueflows/valueflows/pull/392#issuecomment-451946438
> In the usual meanings, although I get confused and so does everybody else but people who do it for a living, "track" means starting with a source and seeing where it went, like starting with a mad cow and see who got parts of it. So it's traversing forwards.

>"Trace" means you got a part of the mad cow and want to find out which animal it came from, what farm, etc. So it is traversing backwards. Provenance is similar to tracing.

I think you two might want to double check which one (track, trace) matches which one (forwards, backwards)

Reply:
So anyway, @elf-pavlik and @fosterlynn ,the definitions in that intro seem to be backwards. Might be my fault....